src/Tools/ExternalAccess/Xaml/Internal/XamlDiagnosticSource.cs

```diff
@@ -0,0 +1,32 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Immutable;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.Diagnostics;
+using Microsoft.CodeAnalysis.LanguageServer;
+using Microsoft.CodeAnalysis.LanguageServer.Handler;
+using Microsoft.CodeAnalysis.LanguageServer.Handler.Diagnostics;
+using Roslyn.LanguageServer.Protocol;
+
+namespace Microsoft.CodeAnalysis.ExternalAccess.Xaml;
+
+internal sealed class XamlDiagnosticSource(IXamlDiagnosticSource xamlDiagnosticSource, TextDocument document) : IDiagnosticSource
+{
+    bool IDiagnosticSource.IsLiveSource() => true;
```

Comment:
nit: we don't generally use explicit interface impl unless necessary.

Reply:
The idea was to avoid polluting PublicAPI.Xxx.txt. I can change it to implicit if needed

Reply:
i see. if it's abotu not being in publicapi, then that's ok.

Reply:
oh, these are the src/Tools/ExternalAccess/Xaml/InternalAPI.Unshipped.txt files?  those don't matter.